kolibri/core/assets/src/views/buttons-and-links/k-external-link.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+
+  <a :class="buttonClasses" :href="href">
+    <!-- Link label text -->
```

Comment:
Replace slot with a label prop.

Reply:
Same thing for the k-router-link

Reply:
Was actually curious why we used a prop instead of a slot on buttons, and was considering switching over those, too.

Seems to artificially make the components less flexible to use props - if there was ever a case where we needed an exception to the style guide rules, makes it harder.

Reply:
> Seems to artificially make the components less flexible to use props
That was the idea, to enforce the rule that only text can be inside buttons.

Reply:
Okay I'll switch it over.

There are trade-offs between forcing vs just encouraging style guide rules. Keeping flexibility is sometimes useful in case we want to intentionally deviate. But we can cross that bridge later.